config.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"io/ioutil"
+
+	"github.com/go-yaml/yaml"
+	"github.com/mitchellh/go-homedir"
+)
+
+type Config struct {
+	IgnoreRules []string `yaml:"ignore_rules"`
+}
+
+func ParseConfig() (*Config, error) {
```

Comment:
__[gometalinter]__
* [interfacer] use of package error not in selector
* [aligncheck] use of package error not in selector
* [structcheck] unused struct field use of package error not in selector
* [varcheck] unused global variable use of package error not in selector